### ISSUE TITLE
Talkback Users are unable to access Pie chart: A11y_.NET 9 template_Home_Screen reader.

### ISSUE THREAD

**Issue by keerthiduvvuri** [dotnet/maui#26909]
### GitHub Tags:
#A11yTCS; #A11ySev1; #Android; #WCAG1.3.1; #SH_.NET9template_Android_Dec2024; #Talkback; #.NET Core; #Device; #A11yMAS; #screen reader; #Element:Piechart; #Info and Relationship; #AINotInscope; 

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
Android Version: 14
Talkback version: 15.1

### Repro Steps:
1. Turn on Talkback and Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Swipe till "Pie chart"
4. Observe that Talkback Users are unable to access Pie chart.

### Actual Result:
Talkback Users are unable to access Pie chart.

### Expected Result:
 Android users should be able to access Pie chart with Talkback.

### User Impact:
Users who relay on screen reader wont get information about Pie chart.

### Attachment:

https://github.com/user-attachments/assets/816cb553-50fb-4b3e-9b03-9c234363d3b3

**Comment by InduPriya1805** [comment]
rev:inse

**Comment by keerthiduvvuri** [comment]
Verified in Latest environment, Issue is Not Fixed. Information present inside of Chart also should be accessible with Talkback.
Added attachment for reference.

https://github.com/user-attachments/assets/5bfd29b0-54ef-477b-926c-793abecde05d

**Comment by keerthiduvvuri** [comment]
Hi @Saravanan-Madhesh Swipe gesture of talkback moves on each and every element on the page, Talkback focus should be moved on each and every element of pie chart and it should announce the information to users.

**Comment by merriemcgaw** [comment]
@PureWeen, @tj-devel709  I'm assuming this is external to MAUI and is part of the way the specific  SyncFusion control is drawn. I seem to recall an Office Hours about this but I don't recall what the specific outcome was. Should we schedule another one to review next steps?

**Comment by keerthiduvvuri** [comment]
Issue is Not fixed in latest environment and Talkback focus is not at all going to Chat section, 
Added attachment for reference.

https://github.com/user-attachments/assets/bb4756f7-467f-4591-b18b-aabb2eb62e5f

**Comment by keerthiduvvuri** [comment]
Closing this bug because issue is Repro in latest version, we are tracking with this https://github.com/dotnet/maui/issues/30921
Hence closing the bug.
